osquery/tables/CMakeLists.txt

```diff
@@ -158,3 +158,17 @@ if(NOT SKIP_LLDPD AND POSIX)
 
   ADD_OSQUERY_LINK_ADDITIONAL("lldpctl")
 endif()
+
+if(NOT SKIP_IPMI AND POSIX)
```

Comment:
Up above you're restricting your view to `LINUX`, but here it's `POSIX`. Should it be `LINUX`?